client/my-sites/controller.js

```diff
@@ -244,6 +245,8 @@ function onSelectedSiteAvailable( context ) {
 		}
 	}
 
+	context.store.dispatch( fetchAutomatedTransferStatus( selectedSite.ID ) );
```

Comment:
I know we need to fetch the status for testing purposes, but we talked about fetching the status only if `hasSitePendingAutomatedTransfer` and probably only when receiving new sites data, right?